in_toto/in_toto_sign.py

```diff
@@ -84,25 +94,47 @@ def _sign_and_dump_metadata(metadata, args):
     if not args.append:
       metadata.signatures = []
 
-    for key_path in args.key:
-      key = util.prompt_import_rsa_key_from_file(key_path)
-      metadata.sign(key)
-
-      # Only relevant when signing Link metadata, where there is only one key
-      keyid = key["keyid"]
-
+    signature = None
+    # If the cli tool was called with `--gpg [KEYID ...]` `args.gpg` is
+    # a list (not None) and we will try to sign with gpg.
+    # If `--gpg-home` was not set, args.gpg_home is None and the signer tries
+    # to use the default gpg keyring.
+    if args.gpg != None:
+      # If `--gpg` was passed without argument we sign with the default key
+      # Excluded so that coverage does not vary in different test environments
+      if len(args.gpg) == 0: # pragma: no cover
+        signature = metadata.sign_gpg(gpg_keyid=None, gpg_home=args.gpg_home)
+
+      # Otherwise we sign with each passed keyid
+      for keyid in args.gpg:
+        # TODO: Add keyid schema check
```

Comment:
Up [here](https://github.com/in-toto/in-toto/pull/150/files#diff-01f3f15afc9e2ce148a2237a8971e849R142) we are using `securesystemslib.schema.KEYID_SCHEMA`. Is this intended?

Reply:
I can add that check too